src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -392,45 +329,81 @@ EmberAfNodeOperationalCertStatus ConvertToNOCResponseStatus(CHIP_ERROR err)
 bool emberAfOperationalCredentialsClusterRemoveAllFabricsCallback(EndpointId endpoint, app::CommandHandler * commandObj)
 {
     emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: Remove all Fabrics");
```

Comment:
I'll add a TODO to remove. This will be removed in the very next PR, that updates the cluster definition. https://github.com/project-chip/connectedhomeip/pull/8826#discussion_r686947790